src/history.js

```diff
@@ -0,0 +1,3 @@
+import { createBrowserHistory } from 'history';
```

Comment:
We could use the default history that always comes with components that have access to the react router dom, this can be achieved by wrapping some containers in `withRouter`.

Reply:
Thanks for that feedback,
am going to look into it and work on the implementation